src/views/Home.vue

```diff
@@ -201,7 +201,77 @@
         </ion-card>
       </div>
       <div v-if="segmentSelected === 'syncStatus'">
```

Comment:
Remove usage of v-if else and toggle the selected segment using id as recommended in ionic8